### ISSUE TITLE
Colours used for "cycleroute" and "30 km/h roads" are too similar

### ISSUE THREAD

**Issue by b33k34** [cyclosm/cyclosm-cartocss-style#495]
Outside London in a small English town without 30km/h speed limits everything is clear 
Here you can see 
- dark blue lines some protected cycleway, 
- pale blue shaded roads - a bit of cycle route (you call it "cycle street”) through town.  
- cyan on the shopping street for 30km/h 
https://www.cyclosm.org/#map=16/51.4318/-0.5418/cyclosm_live

However, this does not work in London because the colours are too similar and laid over the top of each other -  
https://www.cyclosm.org/#map=16/51.4831/-0.1154/cyclosm_live

Every road is now Cyan because all roads in this part of London (and soon all of London) are now 20mph (30km/h) 

There are a lot of roads tagged as ‘cycle route’ (too many - I’m trying to get it cleaned up) but these are on top of cyan roads meaning there is very little contrast.

There are a lot of pink, PRIMARY, roads, again, many of them with blue over the top because they are also tagged as cycle routes. 

The combined effect for me is that it becomes very difficult to pick out the routes and cycleways as everything is quite similar shades of blue and green

**Comment by Florimondable** [comment]
Yes, I already saw that london is difficult to read because of a very dense city with a lot of cycle routes.
Same, issue here in Paris, but that the difficulties of balancing between outdoor cycling and city cycling.

Do you know if those cycle routes do realy exist on the ground ? It seems weird that alwos every three street is in a cycle route.

About the blue colors, do you realy have difficulties to differenciate the blue of Stepney Way the and the street next to it ?
https://www.cyclosm.org/#map=18/51.51675/-0.05192/cyclosm_live
I ask that because I know that people don't see colors the same way.

**Comment by b33k34** [comment]
> On 19 Jan 2021, at 17:33, Florimondable <notifications@github.com> wrote:
> Yes, I already saw that london is difficult to read because of a very dense city with a lot of cycle routes.
> Same, issue here in Paris, but that the difficulties of balancing between outdoor cycling and city cycling.
> 
> Do you know if those cycle routes do realy exist on the ground ? It seems weird that alwos every three street is in a cycle route.
> 

That’s a debate I’m trying to have with the people who have done a lot of the route mapping.  I think the problem is a different (historic) approach to providing for cycling, 

Looking at the Netherlands there are few ‘routes’ within urban areas
https://www.cyclosm.org/#map=14/51.5616/5.0809/cyclosm_live <https://www.cyclosm.org/#map=14/51.5616/5.0809/cyclosm_live>
Because the expectation is that you can cycle everywhere - if a road is busy it will have a protected cycle track and if its a smaller road it will be quiet enough to cycle on.  You dont have to signpost routes you just signpost destinations like you would for a car

But routes do exist between towns where you might otherwise get lost. 
https://www.cyclosm.org/#map=14/51.5342/5.2396/cyclosm_live <https://www.cyclosm.org/#map=14/51.5342/5.2396/cyclosm_live>
 

> Do you know if those cycle routes do realy exist on the ground ? It seems weird that alwos every three street is in a cycle route.
> 

London, historically, tried to work out which roads were a bit quieter than others and direct  cyclists down those - so lots of routes were created between different bits of the city.

But as you say, it seems weird that every few streets is marked as a route.  I don’t think they really are but it seems one very keen mapper has marked all sorts of things as routes - I think a lot could be removed but it’s a big job and theres always a battle to delete things and not get into an edit war where the original mapper who just puts them back. So I’m trying to get work towards cleaning up the routes 

> About the blue colors, do you realy have difficulties to differenciate the blue of Stepney Way the and the street next to it ?
> https://www.cyclosm.org/#map=18/51.51675/-0.05192/cyclosm_live <https://www.cyclosm.org/#map=18/51.51675/-0.05192/cyclosm_live>
> I ask that because I know that people don't see colors the same way.
> 
At that level of zoom I don’t think it’s too difficult to tell them apart, but for some reason I find the cyan really distracting.  

The real problem for me is when zoomed out a bit - to to try to plan a route to a destination.  

Eg https://www.cyclosm.org/#map=15/51.5236/-0.0578/cyclosm_live <https://www.cyclosm.org/#map=15/51.5236/-0.0578/cyclosm_live>

At that level of zoom I find the map problematic. 

Ultimately I think the real issue is that using Cyan for 30km/h roads is lower contrast than the standard of white, and 30km/h is rapidly becoming the default in cities where the map is already ’noisy’. So now urban areas with most detail are less clear than rural areas. 

Eg - s small town outside London where there are few 30km/h roads is much clearer and easier to understand.  
https://www.cyclosm.org/#map=15/51.4166/-0.8246/cyclosm_live <https://www.cyclosm.org/#map=15/51.4166/-0.8246/cyclosm_live>

**Comment by michaelblyons** [comment]
Is this the same as #405?

**Comment by Phyks** [comment]
Indeed, let's keep this one to track this issue. Thanks!

**Comment by reinhard-mueller** [comment]
I think that combining different markers (e.g. for maxspeed and cycling routes) by mixing the colors yields quite unintuitive results.

I propose to indicate cycling routes with a very strong (dark) colored, opaque line in the middle of the highway. Very much like https://www.openstreetmap.org/#&layers=T displays bus lines.
